packages/tools/devtools/devtools-view/package.json

```diff
@@ -44,6 +44,7 @@
 		"build:test:utils": "concurrently npm:build:test:utils:cjs npm:build:test:utils:esm",
 		"build:test:utils:cjs": "fluid-tsc commonjs --project ./src/test/utils/tsconfig.cjs.json",
 		"build:test:utils:esm": "fluid-tsc module --project ./src/test/utils/tsconfig.esm.json",
+		"check:are-the-types-wrong": "attw --pack . --entrypoints .",
```

Comment:
My script adds 'attw' to every package that has the necessary entry points to pass.

This picked up 2 new packages.  It also removed 1 that had no entry points.